appliance_specific/index.md

```diff
@@ -0,0 +1,40 @@
+---
+---
+
+# Backup
+Creating a backup of the appliance will be done using SSH or the console.
+
+1. Launch `appliance_console`
+  a. Select `12) Stop EVM Server Processes` to stop the server processes
+  b. Select `2) Create Database Backup` and choosea backup location (/tmp/evm_db.backup is the default)
+  c. Select `17) Quit` to return to the shell
+2. For convenience, create a TGZ containing all files required to restore the database
+
+        # tar -czvf /root/backup.tgz /tmp/evm_db.backup /var/www/miq/vmdb/GUID /var/www/miq/vmdb/certs/v2_key
```

Comment:
I suggest using \`\`\`bash around this (and then you can remove the comment `#`

Reply:
Also, prefer a 3 space indent (that will allow markdown rendering to put it under the `2.`, but not make it a quote block)